src/core/components/forms/checkboxCard/checkboxCard.tsx

```diff
@@ -40,10 +40,26 @@ export interface ICheckboxCardProps extends ComponentProps<'button'> {
      * Id of the checkbox.
      */
     id?: string;
+    /**
+     * Additional children to render when the checkbox is checked.
+     */
+    children?: ReactNode;
```

Comment:
Can we also make the `description` property optional for the `CheckboxCard` component?